docs/source/release_notes/release-notes.rst

```diff
@@ -31,7 +31,7 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
-develop
+v0.25.0
```

Comment:
I think the general practice is to add `v0.25.0` as a new row, and leave the `develop` one intact (but with an empty list-table).